doc/languages-frameworks/maven.section.md

```diff
@@ -233,7 +233,8 @@ in stdenv.mkDerivation rec {
 
   src = builtins.fetchTarball
     "https://github.com/fzakaria/nixos-maven-example/archive/main.tar.gz";
-  buildInputs = [ maven makeWrapper ];
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ maven ];
```

Comment:
Is that the maven cli? Then it would also go to nativeBuildInputs

Reply:
@fzakaria please check it out

Reply:
should be nativeBuildInputs "technically".
There is no cross-compiling in Java though it's just to be more correct for the graph :)

Reply:
once we enable strictDeps by default then the /bin of packages in buildInputs won't be added to PATH

Reply:
can you do the moving of maven to nativeBuildInputs in a separate PR?